todo_app/data/session_items.py

```diff
@@ -1,41 +1,30 @@
+import os
 from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
+from todo_app.data.api_client import get_cards_request, create_new_card, move_card_to_list
+from todo_app.data.card import Card
 
 def get_items():
-    return session.get('items', _DEFAULT_ITEMS.copy())
-
-def update_task_status(id):
-    item = get_item(id)
-    if (item["status"] == "Not Started"):
-        item["status"] = "Complete"
-    else:
-        item["status"] = "Not Started"
-    save_item(item)
-
-def get_item(id):
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
-
-def add_item(title):
-    items = get_items()
-
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
-    items.append(item)
-    session['items'] = items
-
-    return item
-
-def save_item(item):
-    existing_items = get_items()
-    updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
-    session['items'] = updated_items
-
-    return item
+    cards_result = get_cards_request()
+    card_array = []
+    for list in cards_result:
+            for card in list["cards"]:
+                card_array.append(
+                    Card.from_trello_card(card, list)
+                )
+    return card_array
+
+def update_task_status(card_id, card_status):
+    next_list_id = get_next_list_id(card_status)
+    move_card_to_list(card_id, next_list_id)
+
+def add_item(card_name):
+    create_new_card(card_name)
+
+def get_next_list_id(card_status):
+    match card_status:
+        case "To Do":
```

Comment:
Minor, but note that this assumes the lists have specific names which isn't documented in your Readme assumptions. That seems like a reasonable assumption to me if clarified, but alternatively we could have the items also know their list ids so that we can use that to determine the next list id